meson.build

```diff
@@ -88,10 +88,14 @@ if is_unixy
   dep_wayland_client = dependency('wayland-client',
       required: get_option('with_wayland'), version : '>=1.11')
   dbus_dep = dependency('dbus-1', required: get_option('with_dbus')).partial_dependency(compile_args : true, includes : true)
+  libdrm_dep = dependency('libdrm', required: true)
+  libdrm_amdgpu_dep = dependency('libdrm_amdgpu', required: true)
```

Comment:
Nit: drop the `required: true` - it's the default.

Also it was me, I would squash all the commits into one. As-is things are quite messy.

HTH o/